plugin.yaml

```diff
@@ -147,6 +147,22 @@ node_types:
             cloudstack_config:
                 default: {}
 
+    cloudify.cloudstack.nodes.Volume:
+            derived_from: cloudify.nodes.Volume
+            properties:
+                name:
+                    default: ''
```

Comment:
is this valid? maybe this should not have a default value, so there will be validation when parsing the dsl.

same goes for size